amber-slsa-buildtype.json

```diff
@@ -0,0 +1,137 @@
+{
```

Comment:
Could you add more details either here or to the commit message? I am unsure who / what is expected to consume this file at the moment, or is it just for documentation?